```
Error running mint automation
+ mkdir -p 10175-0cba088/gopath/src/github.com/minio
+ git -C 10175-0cba088/gopath/src/github.com/minio clone --quiet https://github.com/minio/minio.git
+ git -C 10175-0cba088/gopath/src/github.com/minio/minio remote add minio https://github.com/minio/minio.git
+ git -C 10175-0cba088/gopath/src/github.com/minio/minio fetch --quiet minio pull/10175/head:pr10175
+ git -C 10175-0cba088/gopath/src/github.com/minio/minio checkout --quiet pr10175
+ GO111MODULE=on
+ GOPATH=/home/mint-auto/mint-auto/10175-0cba088/gopath
+ make -C 10175-0cba088/gopath/src/github.com/minio/minio --quiet
go: downloading github.com/minio/cli v1.22.0
go: downloading google.golang.org/api v0.5.0
go: downloading github.com/minio/minio-go/v7 v7.0.1
go: downloading gopkg.in/jcmturner/gokrb5.v7 v7.3.0
go: downloading github.com/minio/sha256-simd v0.1.1
go: downloading github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/colinmarc/hdfs/v2 v2.1.1
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading github.com/klauspost/readahead v1.3.1
go: extracting github.com/klauspost/readahead v1.3.1
go: extracting github.com/dustin/go-humanize v1.0.0
go: downloading github.com/klauspost/compress v1.10.3
go: extracting github.com/minio/sha256-simd v0.1.1
go: extracting github.com/minio/cli v1.22.0
go: extracting github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: downloading github.com/Azure/azure-storage-blob-go v0.8.0
go: downloading github.com/tinylib/msgp v1.1.2
go: extracting github.com/colinmarc/hdfs/v2 v2.1.1
go: extracting github.com/tinylib/msgp v1.1.2
go: extracting github.com/minio/minio-go/v7 v7.0.1
go: extracting github.com/Azure/azure-storage-blob-go v0.8.0
go: downloading github.com/shirou/gopsutil v2.20.3-0.20200314133625-53cec6b37e6a+incompatible
go: downloading github.com/cespare/xxhash/v2 v2.1.1
go: downloading github.com/klauspost/reedsolomon v1.9.9
go: downloading github.com/minio/sio v0.2.0
go: downloading github.com/willf/bloom v2.0.3+incompatible
go: extracting github.com/cespare/xxhash/v2 v2.1.1
go: downloading github.com/Azure/azure-pipeline-go v0.2.1
go: extracting github.com/shirou/gopsutil v2.20.3-0.20200314133625-53cec6b37e6a+incompatible
go: extracting github.com/willf/bloom v2.0.3+incompatible
go: extracting github.com/Azure/azure-pipeline-go v0.2.1
go: extracting github.com/minio/sio v0.2.0
go: downloading github.com/gorilla/rpc v1.2.0
go: downloading github.com/minio/highwayhash v1.0.0
go: extracting github.com/klauspost/reedsolomon v1.9.9
go: downloading github.com/cheggaaa/pb v1.0.28
go: extracting github.com/minio/highwayhash v1.0.0
go: extracting github.com/gorilla/rpc v1.2.0
go: extracting github.com/cheggaaa/pb v1.0.28
go: downloading golang.org/x/crypto v0.0.0-20200709230013-948cd5f35899
go: downloading golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae
go: downloading github.com/dchest/siphash v1.2.1
go: downloading gopkg.in/yaml.v2 v2.2.8
go: extracting github.com/dchest/siphash v1.2.1
go: downloading github.com/willf/bitset v1.1.10
go: extracting gopkg.in/yaml.v2 v2.2.8
go: downloading github.com/minio/md5-simd v1.1.0
go: extracting github.com/willf/bitset v1.1.10
go: downloading github.com/mitchellh/go-homedir v1.1.0
go: extracting github.com/minio/md5-simd v1.1.0
go: downloading github.com/ncw/directio v1.0.5
go: extracting github.com/mitchellh/go-homedir v1.1.0
go: downloading gopkg.in/olivere/elastic.v5 v5.0.80
go: extracting golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae
go: extracting github.com/ncw/directio v1.0.5
go: extracting golang.org/x/crypto v0.0.0-20200709230013-948cd5f35899
go: downloading github.com/fatih/structs v1.1.0
go: extracting github.com/fatih/structs v1.1.0
go: downloading github.com/miekg/dns v1.1.8
go: extracting gopkg.in/olivere/elastic.v5 v5.0.80
go: extracting github.com/miekg/dns v1.1.8
go: downloading github.com/streadway/amqp v0.0.0-20190404075320-75d898a42a94
go: downloading github.com/Shopify/sarama v1.24.1
go: downloading github.com/eclipse/paho.mqtt.golang v1.2.0
go: downloading github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: extracting github.com/streadway/amqp v0.0.0-20190404075320-75d898a42a94
go: downloading github.com/nats-io/nats.go v1.10.0
go: extracting github.com/Shopify/sarama v1.24.1
go: extracting github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: extracting github.com/eclipse/paho.mqtt.golang v1.2.0
go: downloading github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: downloading github.com/gorilla/handlers v1.4.2
go: downloading github.com/pierrec/lz4 v2.4.0+incompatible
go: extracting github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: extracting github.com/nats-io/nats.go v1.10.0
go: downloading go.etcd.io/etcd/v3 v3.3.0-rc.0.0.20200707003333-58bb8ae09f8e
go: extracting github.com/gorilla/handlers v1.4.2
go: downloading gopkg.in/ldap.v3 v3.0.3
go: extracting gopkg.in/ldap.v3 v3.0.3
go: downloading github.com/mattn/go-ieproxy v0.0.1
go: extracting github.com/mattn/go-ieproxy v0.0.1
go: downloading github.com/minio/simdjson-go v0.1.5-0.20200303142138-b17fe061ea37
go: extracting github.com/minio/simdjson-go v0.1.5-0.20200303142138-b17fe061ea37
go: downloading github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: extracting github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: downloading github.com/nats-io/jwt v0.3.2
go: extracting github.com/nats-io/jwt v0.3.2
go: downloading github.com/mattn/go-runewidth v0.0.2
go: extracting github.com/mattn/go-runewidth v0.0.2
go: downloading github.com/json-iterator/go v1.1.10
go: extracting github.com/json-iterator/go v1.1.10
go: downloading golang.org/x/net v0.0.0-20200707034311-ab3426394381
go: extracting go.etcd.io/etcd/v3 v3.3.0-rc.0.0.20200707003333-58bb8ae09f8e
go: extracting golang.org/x/net v0.0.0-20200707034311-ab3426394381
go: downloading github.com/elazarl/go-bindata-assetfs v1.0.0
go: extracting github.com/elazarl/go-bindata-assetfs v1.0.0
go: downloading google.golang.org/grpc v1.26.0
go: extracting google.golang.org/grpc v1.26.0
go: downloading github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: extracting github.com/klauspost/compress v1.10.3
go: extracting github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: downloading github.com/philhofer/fwd v1.0.0
go: extracting github.com/philhofer/fwd v1.0.0
go: downloading github.com/prometheus/client_golang v1.0.0
go: downloading gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: extracting github.com/prometheus/client_golang v1.0.0
go: downloading github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
go: extracting gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: downloading github.com/eapache/go-resiliency v1.1.0
go: extracting github.com/eapache/go-resiliency v1.1.0
go: downloading github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
go: extracting github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
go: downloading google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
go: extracting github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
go: downloading github.com/davecgh/go-spew v1.1.1
go: extracting github.com/davecgh/go-spew v1.1.1
go: downloading github.com/nats-io/nkeys v0.1.4
go: extracting github.com/nats-io/nkeys v0.1.4
go: downloading github.com/klauspost/pgzip v1.2.1
go: extracting github.com/klauspost/pgzip v1.2.1
go: downloading github.com/xdg/stringprep v1.0.0
go: extracting github.com/pierrec/lz4 v2.4.0+incompatible
go: extracting github.com/xdg/stringprep v1.0.0
go: downloading github.com/golang/protobuf v1.4.0
go: downloading github.com/eapache/queue v1.1.0
go: extracting github.com/golang/protobuf v1.4.0
go: downloading github.com/prometheus/common v0.4.1
go: extracting github.com/eapache/queue v1.1.0
go: downloading github.com/djherbis/atime v1.0.0
go: extracting github.com/djherbis/atime v1.0.0
go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: extracting github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: downloading github.com/secure-io/sio-go v0.3.0
go: extracting github.com/prometheus/common v0.4.1
go: extracting github.com/secure-io/sio-go v0.3.0
go: downloading github.com/klauspost/cpuid v1.3.1
go: downloading github.com/gomodule/redigo v2.0.0+incompatible
go: extracting github.com/klauspost/cpuid v1.3.1
go: extracting github.com/gomodule/redigo v2.0.0+incompatible
go: downloading github.com/alecthomas/participle v0.2.1
go: downloading github.com/nsqio/go-nsq v1.0.7
go: extracting github.com/alecthomas/participle v0.2.1
go: extracting github.com/nsqio/go-nsq v1.0.7
go: downloading github.com/spaolacci/murmur3 v1.1.0
go: downloading github.com/beorn7/perks v1.0.0
go: extracting github.com/spaolacci/murmur3 v1.1.0
go: downloading github.com/prometheus/procfs v0.0.8
go: extracting github.com/beorn7/perks v1.0.0
go: downloading github.com/gorilla/mux v1.7.5-0.20200711200521-98cb6bf42e08
go: extracting github.com/prometheus/procfs v0.0.8
go: extracting github.com/gorilla/mux v1.7.5-0.20200711200521-98cb6bf42e08
go: downloading github.com/modern-go/reflect2 v1.0.1
go: downloading golang.org/x/text v0.3.3
go: extracting github.com/modern-go/reflect2 v1.0.1
go: extracting google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
go: downloading github.com/pkg/errors v0.8.1
go: extracting github.com/pkg/errors v0.8.1
go: downloading github.com/rjeczalik/notify v0.9.2
go: extracting github.com/rjeczalik/notify v0.9.2
go: downloading github.com/rs/cors v1.7.0
go: extracting github.com/rs/cors v1.7.0
go: downloading github.com/nats-io/stan.go v0.7.0
go: extracting github.com/nats-io/stan.go v0.7.0
go: downloading github.com/gogo/protobuf v1.3.1
go: extracting golang.org/x/text v0.3.3
go: extracting github.com/gogo/protobuf v1.3.1
go: downloading github.com/montanaflynn/stats v0.5.0
go: extracting github.com/montanaflynn/stats v0.5.0
go: extracting google.golang.org/api v0.5.0
go: downloading github.com/google/uuid v1.1.1
go: downloading github.com/fatih/color v1.7.0
go: extracting github.com/fatih/color v1.7.0
go: extracting github.com/google/uuid v1.1.1
go: downloading google.golang.org/protobuf v1.22.0
go: downloading github.com/go-sql-driver/mysql v1.5.0
go: extracting github.com/go-sql-driver/mysql v1.5.0
go: downloading github.com/minio/selfupdate v0.3.1
go: extracting github.com/minio/selfupdate v0.3.1
go: downloading go.uber.org/zap v1.14.1
go: extracting go.uber.org/zap v1.14.1
go: extracting google.golang.org/protobuf v1.22.0
go: downloading github.com/beevik/ntp v0.2.0
go: extracting github.com/beevik/ntp v0.2.0
go: downloading cloud.google.com/go v0.39.0
go: extracting cloud.google.com/go v0.39.0
go: downloading github.com/googleapis/gax-go/v2 v2.0.4
go: extracting github.com/googleapis/gax-go/v2 v2.0.4
go: downloading golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
go: extracting golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
go: extracting gopkg.in/jcmturner/gokrb5.v7 v7.3.0
go: downloading github.com/hashicorp/vault/api v1.0.4
go: downloading go.uber.org/atomic v1.6.0
go: downloading github.com/hashicorp/go-uuid v1.0.1
go: downloading gopkg.in/ini.v1 v1.57.0
go: downloading github.com/coreos/go-semver v0.2.0
go: downloading git.apache.org/thrift.git v0.13.0
go: downloading github.com/mattn/go-isatty v0.0.8
go: downloading github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: downloading github.com/nats-io/nuid v1.0.1
go: extracting github.com/hashicorp/go-uuid v1.0.1
go: downloading github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: downloading go.opencensus.io v0.21.0
go: extracting github.com/mattn/go-isatty v0.0.8
go: extracting go.uber.org/atomic v1.6.0
go: extracting github.com/coreos/go-semver v0.2.0
go: downloading gopkg.in/jcmturner/aescts.v1 v1.0.1
go: downloading github.com/coreos/go-systemd/v22 v22.0.0
go: extracting github.com/hashicorp/vault/api v1.0.4
go: downloading github.com/mattn/go-colorable v0.1.4
go: extracting github.com/nats-io/nuid v1.0.1
go: extracting github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: extracting github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: extracting gopkg.in/ini.v1 v1.57.0
go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.1
go: downloading gopkg.in/jcmturner/rpc.v1 v1.1.0
go: downloading github.com/golang/snappy v0.0.1
go: downloading github.com/hashicorp/hcl v1.0.0
go: downloading go.uber.org/multierr v1.5.0
go: extracting gopkg.in/jcmturner/aescts.v1 v1.0.1
go: downloading github.com/hashicorp/vault/sdk v0.1.13
go: extracting github.com/mattn/go-colorable v0.1.4
go: downloading github.com/coredns/coredns v1.4.0
go: extracting github.com/matttproud/golang_protobuf_extensions v1.0.1
go: extracting github.com/coreos/go-systemd/v22 v22.0.0
go: extracting gopkg.in/jcmturner/rpc.v1 v1.1.0
go: extracting github.com/golang/snappy v0.0.1
go: extracting go.uber.org/multierr v1.5.0
go: extracting go.opencensus.io v0.21.0
go: downloading github.com/hashicorp/go-multierror v1.0.0
go: downloading gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: extracting github.com/hashicorp/hcl v1.0.0
go: downloading github.com/lib/pq v1.7.0
go: downloading github.com/mitchellh/mapstructure v1.1.2
go: downloading github.com/tidwall/gjson v1.3.5
go: extracting github.com/hashicorp/vault/sdk v0.1.13
go: downloading gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: downloading github.com/hashicorp/go-rootcerts v1.0.1
go: downloading github.com/ryanuber/go-glob v1.0.0
go: extracting gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: extracting github.com/hashicorp/go-multierror v1.0.0
go: downloading golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: downloading github.com/tidwall/sjson v1.0.4
go: extracting github.com/coredns/coredns v1.4.0
go: extracting github.com/tidwall/gjson v1.3.5
go: extracting github.com/mitchellh/mapstructure v1.1.2
go: downloading github.com/tidwall/pretty v1.0.0
go: downloading gopkg.in/square/go-jose.v2 v2.3.1
go: extracting github.com/lib/pq v1.7.0
go: extracting github.com/ryanuber/go-glob v1.0.0
go: extracting golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: extracting gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: extracting github.com/tidwall/sjson v1.0.4
go: extracting github.com/hashicorp/go-rootcerts v1.0.1
go: downloading github.com/hashicorp/errwrap v1.0.0
go: downloading github.com/tidwall/match v1.0.1
go: downloading github.com/hashicorp/go-sockaddr v1.0.2
go: downloading github.com/hashicorp/go-retryablehttp v0.5.4
go: downloading github.com/hashicorp/golang-lru v0.5.1
go: downloading github.com/hashicorp/go-cleanhttp v0.5.1
go: extracting github.com/tidwall/pretty v1.0.0
go: extracting github.com/hashicorp/go-cleanhttp v0.5.1
go: extracting github.com/hashicorp/golang-lru v0.5.1
go: extracting github.com/hashicorp/go-retryablehttp v0.5.4
go: extracting github.com/tidwall/match v1.0.1
go: extracting github.com/hashicorp/errwrap v1.0.0
go: extracting github.com/hashicorp/go-sockaddr v1.0.2
go: extracting gopkg.in/square/go-jose.v2 v2.3.1
go: extracting git.apache.org/thrift.git v0.13.0
go: finding gopkg.in/jcmturner/gokrb5.v7 v7.3.0
go: finding google.golang.org/api v0.5.0
go: finding github.com/minio/minio-go/v7 v7.0.1
go: finding github.com/colinmarc/hdfs/v2 v2.1.1
go: finding github.com/minio/sha256-simd v0.1.1
go: finding github.com/minio/cli v1.22.0
go: finding github.com/dustin/go-humanize v1.0.0
go: finding github.com/Azure/azure-pipeline-go v0.2.1
go: finding github.com/dgrijalva/jwt-go v3.2.0+incompatible
go: finding github.com/json-iterator/go v1.1.10
go: finding github.com/minio/highwayhash v1.0.0
go: finding github.com/Azure/azure-storage-blob-go v0.8.0
go: finding github.com/gorilla/mux v1.7.5-0.20200711200521-98cb6bf42e08
go: finding cloud.google.com/go v0.39.0
go: finding golang.org/x/oauth2 v0.0.0-20190226205417-e64efc72b421
go: finding gopkg.in/jcmturner/dnsutils.v1 v1.0.1
go: finding github.com/jcmturner/gofork v0.0.0-20190328161633-dc7c13fece03
go: finding github.com/montanaflynn/stats v0.5.0
go: finding github.com/hashicorp/go-uuid v1.0.1
go: finding github.com/fatih/color v1.7.0
go: finding github.com/mattn/go-isatty v0.0.8
go: finding github.com/mattn/go-colorable v0.1.4
go: finding github.com/mattn/go-ieproxy v0.0.1
go: finding google.golang.org/grpc v1.26.0
go: finding golang.org/x/sys v0.0.0-20200625212154-ddb9806d33ae
go: finding github.com/golang/protobuf v1.4.0
go: finding golang.org/x/crypto v0.0.0-20200709230013-948cd5f35899
go: finding github.com/modern-go/reflect2 v1.0.1
go: finding google.golang.org/genproto v0.0.0-20190819201941-24fa4b261c55
go: finding github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
go: finding github.com/rjeczalik/notify v0.9.2
go: finding github.com/valyala/tcplisten v0.0.0-20161114210144-ceec8f93295a
go: finding github.com/mitchellh/go-homedir v1.1.0
go: finding gopkg.in/ini.v1 v1.57.0
go: finding github.com/googleapis/gax-go/v2 v2.0.4
go: finding go.opencensus.io v0.21.0
go: finding github.com/secure-io/sio-go v0.3.0
go: finding github.com/google/uuid v1.1.1
go: finding github.com/ncw/directio v1.0.5
go: finding google.golang.org/protobuf v1.22.0
go: finding github.com/shirou/gopsutil v2.20.3-0.20200314133625-53cec6b37e6a+incompatible
go: finding github.com/minio/md5-simd v1.1.0
go: finding golang.org/x/net v0.0.0-20200707034311-ab3426394381
go: finding github.com/hashicorp/golang-lru v0.5.1
go: finding github.com/klauspost/cpuid v1.3.1
go: finding gopkg.in/jcmturner/goidentity.v3 v3.0.0
go: finding gopkg.in/jcmturner/aescts.v1 v1.0.1
go: finding golang.org/x/text v0.3.3
go: finding gopkg.in/jcmturner/rpc.v1 v1.1.0
go: finding github.com/cespare/xxhash/v2 v2.1.1
go: finding github.com/cheggaaa/pb v1.0.28
go: finding github.com/mattn/go-runewidth v0.0.2
go: finding github.com/dchest/siphash v1.2.1
go: finding github.com/djherbis/atime v1.0.0
go: finding github.com/elazarl/go-bindata-assetfs v1.0.0
go: finding github.com/gorilla/handlers v1.4.2
go: finding github.com/gorilla/rpc v1.2.0
go: finding github.com/klauspost/compress v1.10.3
go: finding github.com/klauspost/readahead v1.3.1
go: finding github.com/klauspost/reedsolomon v1.9.9
go: finding github.com/miekg/dns v1.1.8
go: finding go.etcd.io/etcd/v3 v3.3.0-rc.0.0.20200707003333-58bb8ae09f8e
go: finding github.com/gogo/protobuf v1.3.1
go: finding go.uber.org/zap v1.14.1
go: finding go.uber.org/atomic v1.6.0
go: finding go.uber.org/multierr v1.5.0
go: finding github.com/coreos/go-systemd/v22 v22.0.0
go: finding github.com/coreos/go-semver v0.2.0
go: finding github.com/coredns/coredns v1.4.0
go: finding gopkg.in/ldap.v3 v3.0.3
go: finding gopkg.in/asn1-ber.v1 v1.0.0-20181015200546-f715ec2f112d
go: finding github.com/Shopify/sarama v1.24.1
go: finding github.com/davecgh/go-spew v1.1.1
go: finding github.com/eapache/go-resiliency v1.1.0
go: finding github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
go: finding github.com/golang/snappy v0.0.1
go: finding github.com/eapache/queue v1.1.0
go: finding github.com/pierrec/lz4 v2.4.0+incompatible
go: finding github.com/rcrowley/go-metrics v0.0.0-20181016184325-3113b8401b8a
go: finding github.com/eclipse/paho.mqtt.golang v1.2.0
go: finding github.com/go-sql-driver/mysql v1.5.0
go: finding github.com/gomodule/redigo v2.0.0+incompatible
go: finding github.com/lib/pq v1.7.0
go: finding github.com/nats-io/nats.go v1.10.0
go: finding github.com/nats-io/jwt v0.3.2
go: finding github.com/nats-io/nkeys v0.1.4
go: finding github.com/nats-io/nuid v1.0.1
go: finding github.com/nats-io/stan.go v0.7.0
go: finding github.com/nsqio/go-nsq v1.0.7
go: finding github.com/pkg/errors v0.8.1
go: finding github.com/streadway/amqp v0.0.0-20190404075320-75d898a42a94
go: finding github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
go: finding github.com/xdg/stringprep v1.0.0
go: finding gopkg.in/olivere/elastic.v5 v5.0.80
go: finding github.com/mailru/easyjson v0.0.0-20180730094502-03f2033d19d5
go: finding github.com/hashicorp/vault/api v1.0.4
go: finding github.com/hashicorp/errwrap v1.0.0
go: finding github.com/hashicorp/go-cleanhttp v0.5.1
go: finding github.com/hashicorp/go-multierror v1.0.0
go: finding github.com/hashicorp/go-retryablehttp v0.5.4
go: finding github.com/hashicorp/go-rootcerts v1.0.1
go: finding github.com/hashicorp/hcl v1.0.0
go: finding github.com/hashicorp/vault/sdk v0.1.13
go: finding github.com/hashicorp/go-sockaddr v1.0.2
go: finding github.com/ryanuber/go-glob v1.0.0
go: finding github.com/mitchellh/mapstructure v1.1.2
go: finding golang.org/x/time v0.0.0-20190308202827-9d24e82272b4
go: finding gopkg.in/square/go-jose.v2 v2.3.1
go: finding github.com/minio/sio v0.2.0
go: finding github.com/beevik/ntp v0.2.0
go: finding github.com/fatih/structs v1.1.0
go: finding gopkg.in/yaml.v2 v2.2.8
go: finding github.com/klauspost/pgzip v1.2.1
go: finding github.com/bcicen/jstream v0.0.0-20190220045926-16c1f8af81c2
go: finding github.com/alecthomas/participle v0.2.1
go: finding github.com/minio/simdjson-go v0.1.5-0.20200303142138-b17fe061ea37
go: finding git.apache.org/thrift.git v0.13.0
go: finding github.com/tidwall/gjson v1.3.5
go: finding github.com/tidwall/match v1.0.1
go: finding github.com/tidwall/pretty v1.0.0
go: finding github.com/tidwall/sjson v1.0.4
go: finding github.com/minio/selfupdate v0.3.1
go: finding github.com/prometheus/client_golang v1.0.0
go: finding github.com/beorn7/perks v1.0.0
go: finding github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
go: finding github.com/prometheus/common v0.4.1
go: finding github.com/matttproud/golang_protobuf_extensions v1.0.1
go: finding github.com/prometheus/procfs v0.0.8
go: finding github.com/rs/cors v1.7.0
go: finding github.com/tinylib/msgp v1.1.2
go: finding github.com/philhofer/fwd v1.0.0
go: finding github.com/willf/bloom v2.0.3+incompatible
go: finding github.com/spaolacci/murmur3 v1.1.0
go: finding github.com/willf/bitset v1.1.10
# github.com/minio/minio/cmd
cmd/erasure-sets.go:1192:11: sErrs declared and not used
make: *** [build] Error 2

```

--- a/cmd/config/errors.go
+++ b/cmd/config/errors.go
@@ -187,12 +187,6 @@ Example 1:
 		"",
 	)
 
-	ErrCorruptedBackend = newErrFn(
-		"Unable to use the specified backend, pre-existing content detected",
-		"Please ensure your disk mount does not have any pre-existing content",
-		"",
-	)
-
 	ErrUnableToWriteInBackend = newErrFn(
 		"Unable to write to the backend",
 		"Please ensure MinIO binary has write permissions for the backend",

--- a/cmd/erasure-healing.go
+++ b/cmd/erasure-healing.go
@@ -18,6 +18,7 @@ package cmd
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"io"
 	"sync"
@@ -197,10 +198,10 @@ func listAllBuckets(storageDisks []StorageAPI, healBuckets map[string]VolInfo) (
 // Only heal on disks where we are sure that healing is needed. We can expand
 // this list as and when we figure out more errors can be added to this list safely.
 func shouldHealObjectOnDisk(erErr, dataErr error, meta FileInfo, quorumModTime time.Time) bool {
-	switch erErr {
-	case errFileNotFound, errFileVersionNotFound:
+	switch {
+	case errors.Is(erErr, errFileNotFound) || errors.Is(erErr, errFileVersionNotFound):
 		return true
-	case errCorruptedFormat:
+	case errors.Is(erErr, errCorruptedFormat):
 		return true
 	}
 	if erErr == nil {
@@ -686,9 +687,9 @@ func isObjectDangling(metaArr []FileInfo, errs []error, dataErrs []error) (valid
 	// or when er.meta is not readable in read quorum disks.
 	var notFoundErasureMeta, corruptedErasureMeta int
 	for _, readErr := range errs {
-		if readErr == errFileNotFound || readErr == errFileVersionNotFound {
+		if errors.Is(readErr, errFileNotFound) || errors.Is(readErr, errFileVersionNotFound) {
 			notFoundErasureMeta++
-		} else if readErr == errCorruptedFormat {
+		} else if errors.Is(readErr, errCorruptedFormat) {
 			corruptedErasureMeta++
 		}
 	}
@@ -699,7 +700,10 @@ func isObjectDangling(metaArr []FileInfo, errs []error, dataErrs []error) (valid
 		// double counting when both parts and er.meta
 		// are not available.
 		if errs[i] != dataErrs[i] {
-			if dataErrs[i] == errFileNotFound || dataErrs[i] == errFileVersionNotFound {
+			if IsErr(dataErrs[i], []error{
+				errFileNotFound,
+				errFileVersionNotFound,
+			}...) {
 				notFoundParts++
 			}
 		}

--- a/cmd/erasure-sets.go
+++ b/cmd/erasure-sets.go
@@ -1199,19 +1199,9 @@ func (s *erasureSets) ReloadFormat(ctx context.Context, dryRun bool) (err error)
 		}
 	}(storageDisks)
 
-	formats, sErrs := loadFormatErasureAll(storageDisks, false)
+	formats, _ := loadFormatErasureAll(storageDisks, false)
 	if err = checkFormatErasureValues(formats, s.drivesPerSet); err != nil {
 		return err
-	}
-
-	for index, sErr := range sErrs {
-		if sErr != nil {
-			// Look for acceptable heal errors, for any other
-			// errors we should simply quit and return.
-			if _, ok := formatHealErrors[sErr]; !ok {
-				return fmt.Errorf("Disk %s: %w", s.endpoints[index], sErr)
-			}
-		}
 	}
 
 	refFormat, err := getFormatErasureInQuorum(formats)
@@ -1355,16 +1345,6 @@ func (s *erasureSets) HealFormat(ctx context.Context, dryRun bool) (res madmin.H
 	for k, v := range beforeDrives {
 		res.Before.Drives[k] = madmin.HealDriveInfo(v)
 		res.After.Drives[k] = madmin.HealDriveInfo(v)
-	}
-
-	for index, sErr := range sErrs {
-		if sErr != nil {
-			// Look for acceptable heal errors, for any other
-			// errors we should simply quit and return.
-			if _, ok := formatHealErrors[sErr]; !ok {
-				return res, fmt.Errorf("Disk %s: %w", s.endpoints[index], sErr)
-			}
-		}
 	}
 
 	if countErrs(sErrs, errUnformattedDisk) == 0 {

--- a/cmd/erasure.go
+++ b/cmd/erasure.go
@@ -18,6 +18,7 @@ package cmd
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"sort"
 	"sync"
@@ -89,18 +90,18 @@ func (d byDiskTotal) Less(i, j int) bool {
 
 func diskErrToDriveState(err error) (state string) {
 	state = madmin.DriveStateUnknown
-	switch err {
-	case errDiskNotFound:
+	switch {
+	case errors.Is(err, errDiskNotFound):
 		state = madmin.DriveStateOffline
-	case errCorruptedFormat:
+	case errors.Is(err, errCorruptedFormat):
 		state = madmin.DriveStateCorrupt
-	case errUnformattedDisk:
+	case errors.Is(err, errUnformattedDisk):
 		state = madmin.DriveStateUnformatted
-	case errDiskAccessDenied:
+	case errors.Is(err, errDiskAccessDenied):
 		state = madmin.DriveStatePermission
-	case errFaultyDisk:
+	case errors.Is(err, errFaultyDisk):
 		state = madmin.DriveStateFaulty
-	case nil:
+	case err == nil:
 		state = madmin.DriveStateOk
 	}
 	return

--- a/cmd/format-erasure.go
+++ b/cmd/format-erasure.go
@@ -27,7 +27,6 @@ import (
 	"sync"
 
 	humanize "github.com/dustin/go-humanize"
-	"github.com/minio/minio/cmd/config"
 	"github.com/minio/minio/cmd/config/storageclass"
 	"github.com/minio/minio/cmd/logger"
 	"github.com/minio/minio/pkg/color"
@@ -57,18 +56,6 @@ const (
 
 // Offline disk UUID represents an offline disk.
 const offlineDiskUUID = "ffffffff-ffff-ffff-ffff-ffffffffffff"
-
-// Healing is only supported for the list of errors mentioned here.
-var formatHealErrors = map[error]struct{}{
-	errUnformattedDisk: {},
-	errDiskNotFound:    {},
-}
-
-// List of errors considered critical for disk formatting.
-var formatCriticalErrors = map[error]struct{}{
-	errCorruptedFormat: {},
-	errFaultyDisk:      {},
-}
 
 // Used to detect the version of "xl" format.
 type formatErasureVersionDetect struct {
@@ -415,7 +402,8 @@ func loadFormatErasure(disk StorageAPI) (format *formatErasureV3, err error) {
 			}
 			if !isHiddenDirectories(vols...) {
 				// 'format.json' not found, but we found user data, reject such disks.
-				return nil, errCorruptedFormat
+				return nil, fmt.Errorf("some unexpected files '%v' found on %s: %w",
+					vols, disk, errCorruptedFormat)
 			}
 			// No other data found, its a fresh disk.
 			return nil, errUnformattedDisk
@@ -490,7 +478,8 @@ func formatErasureGetDeploymentID(refFormat *formatErasureV3, formats []*formatE
 			} else if deploymentID != format.ID {
 				// DeploymentID found earlier doesn't match with the
 				// current format.json's ID.
-				return "", errCorruptedFormat
+				return "", fmt.Errorf("Deployment IDs do not match expected %s, got %s: %w",
+					deploymentID, format.ID, errCorruptedFormat)
 			}
 		}
 	}
@@ -500,14 +489,7 @@ func formatErasureGetDeploymentID(refFormat *formatErasureV3, formats []*formatE
 // formatErasureFixDeploymentID - Add deployment id if it is not present.
 func formatErasureFixDeploymentID(endpoints Endpoints, storageDisks []StorageAPI, refFormat *formatErasureV3) (err error) {
 	// Attempt to load all `format.json` from all disks.
-	var sErrs []error
-	formats, sErrs := loadFormatErasureAll(storageDisks, false)
-	for i, sErr := range sErrs {
-		if _, ok := formatCriticalErrors[sErr]; ok {
-			return config.ErrCorruptedBackend(err).Hint(fmt.Sprintf("Clear any pre-existing content on %s", endpoints[i]))
-		}
-	}
-
+	formats, _ := loadFormatErasureAll(storageDisks, false)
 	for index := range formats {
 		// If the Erasure sets do not match, set those formats to nil,
 		// We do not have to update the ID on those format.json file.
@@ -515,6 +497,7 @@ func formatErasureFixDeploymentID(endpoints Endpoints, storageDisks []StorageAPI
 			formats[index] = nil
 		}
 	}
+
 	refFormat.ID, err = formatErasureGetDeploymentID(refFormat, formats)
 	if err != nil {
 		return err

--- a/cmd/format-erasure_test.go
+++ b/cmd/format-erasure_test.go
@@ -18,6 +18,7 @@ package cmd
 
 import (
 	"encoding/json"
+	"errors"
 	"io/ioutil"
 	"os"
 	"reflect"
@@ -436,8 +437,8 @@ func TestGetErasureID(t *testing.T) {
 	}
 
 	formats[2].ID = "bad-id"
-	if _, err = formatErasureGetDeploymentID(quorumFormat, formats); err != errCorruptedFormat {
-		t.Fatal("Unexpected Success")
+	if _, err = formatErasureGetDeploymentID(quorumFormat, formats); !errors.Is(err, errCorruptedFormat) {
+		t.Fatalf("Unexpect error %s", err)
 	}
 }
 

--- a/cmd/prepare-storage.go
+++ b/cmd/prepare-storage.go
@@ -27,7 +27,6 @@ import (
 	"time"
 
 	"github.com/dustin/go-humanize"
-	"github.com/minio/minio/cmd/config"
 	xhttp "github.com/minio/minio/cmd/http"
 	"github.com/minio/minio/cmd/logger"
 	"github.com/minio/minio/pkg/sync/errgroup"
@@ -253,10 +252,7 @@ func connectLoadInitFormats(retryCount int, firstDisk bool, endpoints Endpoints,
 	formatConfigs, sErrs := loadFormatErasureAll(storageDisks, false)
 	// Check if we have
 	for i, sErr := range sErrs {
-		if _, ok := formatCriticalErrors[sErr]; ok {
-			return nil, nil, config.ErrCorruptedBackend(err).Hint(fmt.Sprintf("Clear any pre-existing content on %s", endpoints[i]))
-		}
-		// not critical error but still print the error, nonetheless, which is perhaps unhandled
+		// print the error, nonetheless, which is perhaps unhandled
 		if sErr != errUnformattedDisk && sErr != errDiskNotFound && retryCount >= 5 {
 			if sErr != nil {
 				logger.Info("Unable to read 'format.json' from %s: %v\n", endpoints[i], sErr)

--- a/cmd/storage-errors.go
+++ b/cmd/storage-errors.go
@@ -19,10 +19,10 @@ package cmd
 import "os"
 
 // errUnexpected - unexpected error, requires manual intervention.
-var errUnexpected = StorageErr("Unexpected error, please report this issue at https://github.com/minio/minio/issues")
+var errUnexpected = StorageErr("unexpected error, please report this issue at https://github.com/minio/minio/issues")
 
 // errCorruptedFormat - corrupted backend format.
-var errCorruptedFormat = StorageErr("corrupted backend format, please join https://slack.min.io for assistance")
+var errCorruptedFormat = StorageErr("corrupted backend format, specified disk mount has unexpected previous content")
 
 // errUnformattedDisk - unformatted disk found.
 var errUnformattedDisk = StorageErr("unformatted disk found")

--- a/cmd/xl-storage.go
+++ b/cmd/xl-storage.go
@@ -505,6 +505,7 @@ func (s *xlStorage) GetDiskID() (string, error) {
 		// Somebody else got the lock first.
 		return diskID, nil
 	}
+
 	formatFile := pathJoin(s.diskPath, minioMetaBucket, formatConfigFile)
 	fi, err := os.Stat(formatFile)
 	if err != nil {
@@ -520,8 +521,12 @@ func (s *xlStorage) GetDiskID() (string, error) {
 			} else if os.IsPermission(err) {
 				return "", errDiskAccessDenied
 			}
-			return "", err
+			logger.LogIf(GlobalContext, err) // log unexpected errors
+			return "", errCorruptedFormat
+		} else if os.IsPermission(err) {
+			return "", errDiskAccessDenied
 		}
+		logger.LogIf(GlobalContext, err) // log unexpected errors
 		return "", errCorruptedFormat
 	}
 
@@ -533,13 +538,34 @@ func (s *xlStorage) GetDiskID() (string, error) {
 
 	b, err := ioutil.ReadFile(formatFile)
 	if err != nil {
+		// If the disk is still not initialized.
+		if os.IsNotExist(err) {
+			_, err = os.Stat(s.diskPath)
+			if err == nil {
+				// Disk is present but missing `format.json`
+				return "", errUnformattedDisk
+			}
+			if os.IsNotExist(err) {
+				return "", errDiskNotFound
+			} else if os.IsPermission(err) {
+				return "", errDiskAccessDenied
+			}
+			logger.LogIf(GlobalContext, err) // log unexpected errors
+			return "", errCorruptedFormat
+		} else if os.IsPermission(err) {
+			return "", errDiskAccessDenied
+		}
+		logger.LogIf(GlobalContext, err) // log unexpected errors
 		return "", errCorruptedFormat
 	}
+
 	format := &formatErasureV3{}
 	var json = jsoniter.ConfigCompatibleWithStandardLibrary
 	if err = json.Unmarshal(b, &format); err != nil {
+		logger.LogIf(GlobalContext, err) // log unexpected errors
 		return "", errCorruptedFormat
 	}
+
 	s.diskID = format.Erasure.This
 	s.formatFileInfo = fi
 	s.formatLastCheck = time.Now()

